### ISSUE TITLE
Installation error on R 3.5.0 Windows

### ISSUE THREAD

**Issue by GjjvdBurg** [r-lib/cli#313]
Hi there,

First of all, thanks for making this package! Unfortunately I'm getting a compilation error on Windows when testing on R 3.5.0. The error is about a missing ``Altrep.h`` header file. From what I can find this error occurred in other packages as well (see [vctrs](https://github.com/r-lib/vctrs/issues/1153), for instance), because the file [wasn't included](https://github.com/rwinlib/base/issues/9) in the R 3.5.0 binary. Here is the compilation log:

```
In R CMD INSTALL
* installing *source* package 'cli' ...
** package 'cli' successfully unpacked and MD5 sums checked
** libs

*** arch - i386
c:/Rtools/mingw_32/bin/gcc  -I"C:/R/include" -DNDEBUG          -O3 -Wall  -std=gnu99 -mtune=generic -c cleancall.c -o cleancall.o
c:/Rtools/mingw_32/bin/gcc  -I"C:/R/include" -DNDEBUG          -O3 -Wall  -std=gnu99 -mtune=generic -c errors.c -o errors.o
c:/Rtools/mingw_32/bin/gcc  -I"C:/R/include" -DNDEBUG          -O3 -Wall  -std=gnu99 -mtune=generic -c init.c -o init.o
c:/Rtools/mingw_32/bin/gcc  -I"C:/R/include" -DNDEBUG          -O3 -Wall  -std=gnu99 -mtune=generic -c progress-altrep.c -o progress-altrep.o
progress-altrep.c:32:26: fatal error: R_ext/Altrep.h: No such file or directory
 #include <R_ext/Altrep.h>
                          ^
compilation terminated.
make: *** [C:/R/etc/i386/Makeconf:208: progress-altrep.o] Error 1
ERROR: compilation failed for package 'cli'
```

If it helps, the full installation log where this occurs for me can be found [here](https://github.com/GjjvdBurg/SyncRNG/runs/3028394524?check_suite_focus=true#step:4:2039). Thanks in advance for your help!

**Comment by gaborcsardi** [comment]
Seems like this is a bug in R 3.5.0, that was fixed in R 3.5.1. Can you update to that?

**Comment by GjjvdBurg** [comment]
Thanks for the quick response! I indeed ended up moving away from R 3.5.0 as a workaround for this issue.
